hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -296,6 +300,42 @@ public boolean isBeforeTimelineStarts(String instant) {
     return details.apply(instant);
   }
 
+  /**
```

Comment:
Timeline APIs are only about instants  in general. I think adding partitions here is breaking that abstraction. Can you move this to some helper class

Reply:
Thank you. Moved it